lib/class-wp-rest-menus-controller.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+/**
+ * Class WP_REST_Menus_Controller
+ */
+class WP_REST_Menus_Controller extends WP_REST_Terms_Controller {
+
+	/**
+	 * Get the term, if the ID is valid.
+	 *
+	 * @param int $id Supplied ID.
+	 *
+	 * @return WP_Term|WP_Error Term object if ID is valid, WP_Error otherwise.
+	 */
+	protected function get_term( $id ) {
+		$error = new WP_Error( 'rest_term_invalid', __( 'Term does not exist.' ), array( 'status' => 404 ) );
```

Comment:
Not sure why it was added in this variable. :/

Reply:
Copied from here - https://github.com/WordPress/WordPress/blob/131440c1a5f0e1e2273f7c2fff2533b94c77c30d/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php#L314-L331